### PR TITLE
Add plugin scaffold

### PR DESCRIPTION
**Pull request by Nurts**
### What was wrong ?

- #3 

### How it was fixed ?

- Added placeholder plugin as described here: https://docs.pretix.eu/en/latest/development/api/placeholder.html#placeholder-registration

### To Do

- [ ] Maybe replace with some static text (instead of the link) for the scaffold.

### CODE REVIEW

**Comment by Nurts** [comment]
Hi @ligi , I am not sure which link you mentioned that should be replaced in #3

**Comment by ligi** [comment]
the main purpose of this plugin to replace a placeholder with a link generated by the alphawallet tool - this is the link I refer to in the issue

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+setup(
+    name='pretix-attestation-placeholder-plugin',
+    version='0.1.0',
+    description='Pretix Ethereum Plugin Developers',
+    long_description=long_description,
+    url='https://github.com/efdevcon/pretix-attestation-placeholder-plugin',
+    author='Pretix Ethereum Plugin Developers',
+    author_email='Your email',
```
Comment (ligi) [inline setup.py:33]:
```suggestion
    author_email='ticketing@devcon.org',
```

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    license='MIT License',
+
+    install_requires=[
+        "pretix>=3.8.0",
```
Comment (ligi) [inline setup.py:37]: in pretix_attestation_plugin/__init__.py we require >=3.7.0 - maybe we should do the same there or here. Let's just unify it - both fine with me

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**